spec/shared_print/deprecation_record_spec.rb

```diff
@@ -0,0 +1,167 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+require "shared_print/deprecation_record"
+
+RSpec.describe SharedPrint::DeprecationRecord do
+  let(:org) { "umich" }
+  let(:org_2) { "smu" }
+  let(:ocn) { 111 }
+  let(:bad_ocn) { -999 }
+  let(:loc) { "i123" }
+  let(:loc_2) { "i222" }
+  let(:sta) { "E" }
+  let(:bad_sta) { "X" }
+  let(:line) { [org, ocn, loc, sta].join("\t") }
+  let(:empty_cluster) { create(:cluster) }
+  let(:dep) { described_class.new(organization: org, ocn: empty_cluster.ocns.first, local_id: loc, status: sta) }
+
+  def make_commitment(ocn, org, local_id)
+    # Make and cluster a commitment
+    com = build(:commitment, ocn: ocn, organization: org, local_id: local_id)
+    cluster = Clustering::ClusterCommitment.new(com).cluster.tap(&:save)
+    cluster.commitments.where(uuid: com.uuid).first
+  end
+
+  before(:each) do
+    Cluster.collection.find.delete_many
+  end
+
+  describe "#initialize" do
+    it "pos" do
+      expect do
+        described_class.new(
+          organization: org,
+          ocn: ocn,
+          local_id: loc,
+          status: sta
+        )
+      end.not_to raise_error
+    end
+
+    it "neg, status" do
```

Comment:
`it "raises ArgumentError when given an unknown status"` (should try to avoid "good" or "bad" value judgments when possible; they're just inputs the program doesn't handle or understand)